telegram/_poll.py

```diff
@@ -28,12 +28,79 @@
 from telegram._utils import enum
 from telegram._utils.argumentparsing import parse_sequence_arg
 from telegram._utils.datetime import extract_tzinfo_from_defaults, from_timestamp
-from telegram._utils.types import JSONDict
+from telegram._utils.defaultvalue import DEFAULT_NONE
+from telegram._utils.types import JSONDict, ODVInput
 
 if TYPE_CHECKING:
     from telegram import Bot
 
 
+class InputPollOption(TelegramObject):
+    """
+    This object contains information about one answer option in a poll to send.
+
+    Objects of this class are comparable in terms of equality. Two objects of this class are
+    considered equal, if their :attr:`text` is equal.
+
+    .. versionadded:: NEXT.VERSION
+
+    Args:
+        text (:obj:`str`): Option text,
+            :tg-const:`telegram.PollOption.MIN_LENGTH`-:tg-const:`telegram.PollOption.MAX_LENGTH`
+            characters.
+        text_parse_mode (:obj:`str`, optional): |parse_mode|
+            Currently, only custom emoji entities are allowed.
+        text_entities (Sequence[:class:`telegram.MessageEntity`], optional): Special entities
+            that appear in the option :paramref:`text`. It can be specified instead of
+            :paramref:`text_parse_mode`.
+            Currently, only custom emoji entities are allowed.
+            This list is empty if the text does not contain entities.
```

Comment:
do we need to mention that it's empty? Isn't it obvious?

Reply:
the alternative would be that it's `None`, but we explicitly make it an empty sequence. We do that for a couple of array-type attributes, see #3057. However, reading that prs description again, I see that I explicitly opted out of doing that for input-type classes. I'll try to double check if we have some problems with this behavior

Reply:
Had another closer look. Since #3249, we have *all* array-type attributes as always-tuple. This does not seem to cause any problem. The same PR made `TO.to_dict()` skip empty sequence arguments, so empty tuples are not reported to TG. I don't know why in #3057 I thought this to be a problem. In any case, never change a running system, so I'll leave it as is :D